lib/sqlalchemy/sql/elements.py

```diff
@@ -1139,10 +1140,6 @@ def __add__(self, other: Any) -> ColumnElement[Any]:
         def __radd__(self: _SQO[_NT], other: Any) -> ColumnElement[_NT]:
             ...
 
-        @overload
-        def __radd__(self: _SQO[int], other: Any) -> ColumnElement[int]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

this is because for some wired reason float is a superclass of int in pep 484, right?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4834

Reply:
I've removed it because running mypy raises:

```
lib\sqlalchemy\sql\elements.py:1143: error: Overloaded function signature 2 will never be matched: signature 1's parameter type(s) are the same or broader  [misc]
```

To avoid this error we can either push it above the first overload, or remove it. But since `_NT` is already a type parameter, `Any + _SQO[int]` already returns `ColumnElement[int]` so no need for this overload I think. Anyway this is kind of out the scope of this PR so maybe I should readd it. Or if you want I can add a test to prove that `__radd__` does return `ColumnElement[int]` without the overload.